File: lc326.cpp
#include<cstdio>
#include<cmath>
class Solution {
public:
    bool static isPowerOfThree(int n) {
		double a=log(n)/log(3);
		printf("%lf\n",a);
		int b=(int)(a+0.5);
		printf("%d\n",b);
		if(abs(a-b)<1e-4) return true;
		else return false ;
	}
};
int main()
{
	Solution::isPowerOfThree(0);
}

File: lc326.py
class Solution(object):
    def isPowerOfThree(self, n):
        if n>0:
            a=math.log(n)/math.log(3.0);
        else :
            a=0.5;
        b=round(a,0)
        if abs(a-b)<1e-10:
            return True;
        else:
            return False;
            
